cargo-dist/src/errors.rs

```diff
@@ -305,6 +305,11 @@ pub enum DistError {
         /// Name of binary
         bin_name: String,
     },
+
+    /// Error during `cargo dist update`
+    #[error("`cargo dist update` failed; the new version isn't in the place we expected")]
```

Comment:
is this the only way it can fail? it may be useful to give more info on "place" e.g. place on the web? place on the local machine?

Reply:
Ooh yeah, good call.